code-challenges.test.js

```diff
@@ -22,9 +22,47 @@ const secretCodeWord2 = "Gobbledygook"
 const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
-
+describe("crypt", () => {
+    let secretCodeWord1 = "Lackadaisical"
+    let secretCodeWord2 = "Gobbledygook"
+    let secretCodeWord3 = "Eccentric"
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it(" takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.", () => {
+
+        //an expect method, nested within the test block, calling on the crypt() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+        expect(crypt(secretCodeWord1)).toEqual("L4ck4d41s1c4l")
+        expect(crypt(secretCodeWord2)).toEqual("G0bbl3dyg00k")
+        expect(crypt(secretCodeWord3)).toEqual("3cc3ntr1c")
+    })
+})
+// got red
```

Comment:
It would be a good idea to just move your test variables inside the test rather than recreating them.